index.bs

```diff
@@ -296,24 +292,24 @@ partial interface NeuralNetworkContext {
     therefore its usage is encouraged from the performance standpoint.
     <pre highlight="js">
     let shape = [1,-1,1,1];
-    return nn.add(
-        nn.mul(
-          nn.reshape(scale, shape),
-          nn.div(
-            nn.sub(input, reshape(mean, shape)),
-            nn.sqrt(nn.add(nn.reshape(variance, shape), nn.constant(epsilon)))
+    return builder.add(
```

Comment:
ideally it would be great to allow chained api for the Operands.
builder.constant(...).add(builder.constant(...))

Reply:
I think this is a good idea. Created https://github.com/webmachinelearning/webnn/issues/106 to follow up.